CONTRIBUTING.md

```diff
@@ -10,11 +10,6 @@ The following are guidelines for contributing to the Knative documentation.
 These are just guidelines, not rules. Use your best judgment, and feel free to
 propose changes to this document in a pull request.
 
-**First off, thanks for taking the time to contribute!**
-
-The following are guidelines for contributing to the Knative documentation.
-These are just guidelines, not rules. Use your best judgment, and feel free to
-propose changes to this document in a pull request.
```

Comment:
Why remove this?

Reply:
@markusthoemmes When I created the PR, **First off, thanks for taking the time to contribute!** block was written twice. So I removed one instance.

Reply:
Yeah, there were two PRs trying to fix this after this PR. Caught one, but the other got through.